.github/workflows/regression-test.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
       env:
         TEST_PROJECT_SA_KEY: ${{ secrets.TEST_PROJECT_SA_KEY }}
     - name: Gradle Run Regression Tests Only
-      run: ./gradlew service:regressionTests -PregressionTestUnderlays=cmssynpuf,aouSR2019q4r4 --scan
+      run: ./gradlew service:regressionTests -PregressionTestUnderlays=cmssynpuf,aouSR2019q4r4 --info --scan
```

Comment:
A comparison of without - https://github.com/DataBiosphere/tanagra/actions/runs/10859538266/job/30139008330#step:8:1

and with - 
https://github.com/DataBiosphere/tanagra/actions/runs/10857889854/job/30135081427#step:8:1

For now keeping it out since all of the extra logs are less about the application tests etc and more about the gradle build process, which is not needed regularly. 
keeping logs concide really helps with debugging